### ISSUE TITLE
Another Bad Game

### ISSUE THREAD

**Issue by Sakibur-Karim** [SCell555/amd-ags-stub#1]
I found another bad game that seems to exhibit the same behavior: Forza Horizon 4 (Steam Version). Unfortunately, this fix doesn't work but I believe FH4 behaves the same way and tries to communicate with AMD driver.

Also wanted to say thank you for the fix, works perfectly for Dishonored 2, was driving me crazy.

**Comment by crzsotona** [comment]
@Sakibur-Karim 
Does it help to disable integrated AMD graphics?

**Comment by Sakibur-Karim** [comment]
Yes, after completely uninstalling & disabling integrated AMD graphics card via bios it works

**Comment by iksmpro** [comment]
Dishonored Death of the Outsider as well.
